_life/renting_legal.md

```diff
@@ -0,0 +1,73 @@
+
+Whenever these indices change, your rent can change. In practice, the IPC increases If they increase, it allows your landlord to increase your rent. Conversely, if they decrease, you are entitled to request a decrease in the rent. In either case, the new rent will be effective at the next term of your contract (usually your move-in date).
+
+:warn: In case of a rent decrease, it is your own responsibility to request the rent decrease, the landlord will not do it for you. You should always request it if applicable. It not only affects your rent, but the one the next tenants will pay. 
+
+### Calculation
+If you receive a notice of rent increase, it is important that you check that it is in accordance with the law. A proper notice must follow a certain format[^increase_format], lest it is deemed invalid. The increase should be broken down into 3 categories:
+- Increase of the reference mortgage rate: in the current conditions, this induces a 3% increase per 0.25% of increase in the mortgage rate[^mortgage_increase]. $\Delta R = \frac{\text{new rate} - \text{old rate}}{0.25} \times 0.03 \times R$
+- Increase of the IPC: 40% of the relative increase of the index can be applied to your rent: $\Delta R = \frac{\text{new index} - \text{old index}}{\text{old index}} \times 0.4 \times R$
+- Increase of landlord costs: this category is less clearly defined and encapsulates all increases in costs that the landlord can claim.
+
+<p align="center">
+  <img src=rent_increase.jpg/>
+</p>
+
+Above is an example of a rent increase notice, following the official format. By applying the formulas above, one can check that the increases in the first 2 categories indeed follow the law: the mortgage rate increased by 0.5 points, i.e. 2*0.25, hence an increase of  2 * 0.03 * 1210 = 72.60.
+The IPC also increased by about 1%, resulting in an increase of (169.3 - 167.5) / 167.5 * 0.4 * 1210 = 5.2 increase.
+
+The [ASLOCA](/www.asloca.ch)—the Swiss tenant association—provides a calculator on their website that applies the same formulas to check if a rent increase is reasonable or not. If the increase does not follow the formulas above, then you should consider challenging it.
+
+## Rent decreases
+Whenever the indices go down (usually the mortgage rate), you are eligible for a rent decrease. This is your right, and your landlord cannot refuse it if done correctly. The calculator of ASLOCA lets you check how much of a decrease you can ask for, and will provide a [template](https://vaud.asloca.ch/sites/vaud.asloca.ch/files/2025-02/01-lettre-bailleur.docxhttps://vaud.asloca.ch/sites/vaud.asloca.ch/files/2025-02/01-lettre-bailleur.docx)letter to send to the landlord. The decrease will take effect at the next term of your contract, and should be received no later than 3 months prior to that date for the request to be deemed valid.
```

Comment:
Nice! Have you or someone you know tried this?

Reply:
Also, is it essentially the same formulae as above? We could make it a `###` subheading instead of a `##` heading.